docs/reference/indices/create-data-stream.asciidoc

```diff
@@ -71,11 +85,3 @@ Data stream names must meet the following criteria:
 will count towards the 255 limit faster)
 --
 
-[[indices-create-data-stream-api-request-body]]
-==== {api-request-body-title}
-
-`timestamp_field`::
-(Required, string) The name of the timestamp field. This field must be present
-in all documents indexed into the data stream and must be of type
-<<date, `date`>> or <<date_nanos, `date_nanos`>>.
```

Comment:
I think that this documentation on `timestamp_field` should be preserved somewhere else, perhaps in the documentation on creating composable templates.